### ISSUE TITLE
Should info/warn on `FROM --platform=$BUILDPLATFORM`

### ISSUE THREAD

**Issue by skyzyx** [hadolint/hadolint#982]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Specifying `FROM --platform=$BUILDPLATFORM` should be flagged as (a) syntactically-valid, but (b) _probably_ not what you want if you're building a multi-platform image.

### Actual behavior

No info/warning on this.

#### Version

```plain
Haskell Dockerfile Linter 2.12.0
```

#### Dockerfile

```Dockerfile
# syntax=docker/dockerfile:1
FROM --platform=$BUILDPLATFORM alpine:3.18
```

#### Output

_None._

### Additional environment details (OS, stack version, etc.)

* macOS 14.2β
* Installed via Homebrew
* Docker version 24.0.6, build ed223bc

**Comment by alexanderbazhenoff** [comment]
Have the same feature request to ignore FROM line, because in our ansible tests we template our `dockerfile.j2` like:

```
FROM {{ item.image }}
```

Is it possible to ignore the whole line or switch to a more relaxed FROM value?
